scrapers/R18.dev.yml

```diff
@@ -0,0 +1,428 @@
+                with: "Unconsciously"
+              - regex: U\*{7}g\b
+                with: "Unwilling"
+              - regex: V\*{5}e\b
+                with: "Violate"
+              - regex: V\*{1}olated\b
+                with: "Violated"
+              - regex: V\*{5}ed\b
+                with: "Violated"
+              - regex: V\*{5}es\b
+                with: "Violates"
+              - regex: V\*{6}e\b
+                with: "Violence"
+              - regex: V\*{5}t\b
+                with: "Violent"
+              - regex: Y\*{8}l\b
+                with: "Young Girl" # Not Sure
+              - regex: Y\*{8}ls\b
+                with: "Young Girls" # Not Sure
+      Studio:
+        Name: maker_name_en
+      Code:
+        selector: dvd_id
+      Tags:
+        Name:
+          selector: categories.#.name_en
+          postProcess: *ppUncensor
+      URL:
+        selector: content_id
+        postProcess:
+          - replace:
+              - regex: ^
+                with: https://r18.dev/videos/vod/movies/detail/-/id=
+                
+  movieScraper:
+    movie:
+      Name:
+        selector: dvd_id
+      Aliases: title_ja
+      Duration: 
+        selector: runtime_mins
+        postProcess:
+          - replace:
+            - regex: $
+              with: ":00"        
+      Date:
+        selector: release_date
+      FrontImage:
+        selector: jacket_full_url
+      Director:
+        selector: directors.#.name_romaji
+      Synopsis:
+        selector: title_en
+        postProcess: *ppUncensor
+      Studio:
+        Name: maker_name_en
+      URL:
+        selector: content_id
+        postProcess:
+          - replace:
+              - regex: ^
+                with: https://r18.dev/videos/vod/movies/detail/-/id=                
+
+  # Used for fragment scraping - the api is a little messy if we don't have the exact ID and need to do a search because the search result api doesn't have all the info we need. 
+  # So we get the ID from the search result page and make another search for it
+  # Note I am not getting aliases yet for performers here as it didn't seem worth it to make that many subqueries. You should hopefully be matching them up with StashDB versions of the performers
+  # or doing a performer search anyways. 
+  # I've also sent in a request to the dev of the site to make their API a little easier to use, so hopefully that goes through and we can have our cake and eat it too. :)
+  sceneSearchIndirect:
+    scene:
+    # Title not on this page, so we need to fetch another page to get it
+      Title:
+        selector: content_id
+        postProcess:
+          - replace:
+            - regex: (.+)
+              with: https://r18.dev/videos/vod/movies/detail/-/combined=$1/json
+          - subScraper:
+              selector: dvd_id       
+      # Code not on this page, so we need to fetch another page to get it. It's the same as title. Surely there must be a way to only query once?
```

Comment:
At the moment there is no way to keep meta from a query. Another query is required, like you do.